docs/src/air-gapped.md

```diff
@@ -1,40 +1,41 @@
 # Managing clusters in a disconnected or air-gapped environment
 
-In a security controlled environment where no direct connection to the
-Internet is allowed, it is necessary to provide all packages needed by
-TPA to complete the deployment. This can be done via a local-repo on
+In a security-controlled environment where no direct connection to the
+Internet is allowed, you need to provide all packages needed by
+TPA to complete the deployment. You can do this by way of `local-repo` on
 each node in the cluster. TPA supports the addition of custom
-repositories on each node via a
-[local-repo](local-repo.md) and the required packages can be downloaded
-using the [download-packages](tpaexec-download-packages.md) command.
+repositories on each node by way of
+[`local-repo`](local-repo.md). You can can download the required packages
+using the [`download-packages`](tpaexec-download-packages.md) command.
```

Comment:
In both of the above uses of `local-repo`, I want to confirm you meant the command and not the words local repository. I've edited it with the assumption that local-repo is a command.

Reply:
local-repo is not a command but --enable-local-repo and --use-local-repo-only are switches that can be applied to configure command to enable the use of local repositories during deployments. the functionality generates a folder named local-repo in the cluster directory and is at the core of air gapped installation.
So local-repo is not a command but carries more meaning than just `local repository`.